### PR TITLE
Season Review

### PR DESCRIPTION
**Pull request by mienaikoe**
- Added a page for Season Review
  - In the future, this may be a way to also add and review stats. I'm tempted to call it "Stats" but I'm on the fence.

### CODE REVIEW

File: client/pages/Team/InspirationCard/SeasonReviewCard.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+.card{
+  background-image: url('/images/circle.jpg');
```
Comment (k3ntako) [inline client/pages/Team/InspirationCard/SeasonReviewCard.css]: It's not a bad picture, but it's not my favorite. It's your call!

I liked these better:
https://unsplash.com/photos/uCMKx2H1Y38
https://unsplash.com/photos/lbLgFFlADrY

File: client/pages/Review/index.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    const monthHeadings = [
+      <th key="name" />,
+      <th key="totals" colSpan="3">Totals</th>
+    ];
+    const dateHeadings = [
+      <th key="name" />,
+      <th key="posit"><FontIcon value="check"/></th>,
+      <th key="negat"><FontIcon value="format_color_reset"/></th>,
+      <th key="flake"><FontIcon value="error"/></th>,
```
Comment (k3ntako) [inline client/pages/Review/index.js]: I think a tooltip would be helpful here.

File: client/pages/Review/index.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+          classes.push(styles.uncounted);
+        }
+        if( ix === 0 ){
+          classes.push(styles.first);
+        }
+        const classesStr = classes.join(" ");
+
+        dateHeadings.push(<th key={event._id} className={classesStr}>
+          <div>{event.datetime.getDate()}</div>
+        </th>);
```
Comment (k3ntako) [inline client/pages/Review/index.js]: I think a tooltip with the event name would be good.

File: client/pages/Team/InspirationCard/SeasonReviewCard.css (hunk 1 above)
Comment (mienaikoe) [reply client/pages/Team/InspirationCard/SeasonReviewCard.css]: I'll do the huddle. Makes more sense.

File: client/pages/Review/index.css [hunk 4]
```diff
@@ -2,87 +2,87 @@
...
+  line-height: 2.6rem;
+  text-align: right;
 }
 
-.table .dates th > * {
-  background-color: var(--palette-blue-grey-700);
-  width: 2.6rem;
-  height: 2.6rem;
+.table .date > * {
+  background-color: var(--palette-light-blue-700);
```
Comment (k3ntako) [inline client/pages/Review/index.css]: I think we should keep the practices a more subtle color.

File: createPeople.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
+// Copy this file to Mighty-Speed-Krush root
+// In terminal, run `node`
+// hit "return/enter"
```
[the remark this answers is not in the document]
Comment (mienaikoe) [reply createPeople.js]: Ohh I didn't realize I committed this

File: client/pages/Review/index.js [hunk 6]
```diff
@@ -130,17 +157,26 @@ class ReviewPage extends React.Component {
         team.members.forEach( (member, ix) => {
           const isAttending = event.is_attending(member);
           const isNotAttending = event.is_not_attending(member);
```
[the remark this answers is not in the document]
Comment (mienaikoe) [reply client/pages/Review/index.js]: true

File: client/pages/Review/index.js [hunk 7]
```diff
@@ -54,21 +54,15 @@ class ReviewPage extends React.Component {
   }
 
   toggleAttending(){
-    this.setState( prevState => {
-      return { attendingVisible: !prevState.attendingVisible }
-    });
+    this.setState( prevState => { attendingVisible: !prevState.attendingVisible });
```
Comment (k3ntako) [inline client/pages/Review/index.js]: sorry this is my fault. I forgot that `=> {}` would be interpreted as a function. We need to add parentheses around the object.

File: client/pages/Review/index.js [hunk 8]
```diff
@@ -54,21 +54,15 @@ class ReviewPage extends React.Component {
...
-      return { notAttendingVisible: !prevState.notAttendingVisible }
-    });
+    this.setState( prevState => { notAttendingVisible: !prevState.notAttendingVisible });
   }
 
   toggleNoResponse(){
-    this.setState( prevState => {
-      return { noResponseVisible: !prevState.noResponseVisible }
-    });
+    this.setState( prevState => { noResponseVisible: !prevState.noResponseVisible });
```
Comment (k3ntako) [inline client/pages/Review/index.js]: see above

File: client/pages/Review/index.js [hunk 9]
```diff
@@ -54,21 +54,15 @@ class ReviewPage extends React.Component {
...
-      return { attendingVisible: !prevState.attendingVisible }
-    });
+    this.setState( prevState => { attendingVisible: !prevState.attendingVisible });
   }
 
   toggleNotAttending(){
-    this.setState( prevState => {
-      return { notAttendingVisible: !prevState.notAttendingVisible }
-    });
+    this.setState( prevState => { notAttendingVisible: !prevState.notAttendingVisible });
```
Comment (k3ntako) [inline client/pages/Review/index.js]: see above

File: client/pages/Review/index.js (hunk 7 above)
Comment (mienaikoe) [reply client/pages/Review/index.js]: I'm just going to turn this back to what it was

**Pull request merged**